api/src/main/java/org/pmiops/workbench/actionaudit/auditors/EgressEventAuditorImpl.kt

```diff
@@ -32,21 +32,12 @@ constructor(
 ) : EgressEventAuditor {
 
     /**
-     * Converts the AoU-chosen runtime name into the actual VM name as reported by Google Cloud's
-     * flow logs. Empirically, an "-m" suffix is added to the VM name, due to Leo team's use
-     * of Dataproc (see https://cloud.google.com/dataproc/docs/concepts/configuring-clusters/metadata).
-     *
-     * TODO(RW-4848): How do egress alerts interact with worker nodes?
+     * Returns the prefix for a user's VM which captures the following situations:
+     * 1. GCE VMs: all-of-us-<user_id>
+     * 2. Dataproc master nodes: all-of-us-<user_id>-m
+     * 3. Dataproc worker nodes: all-of-us-<user_id>-w-<index>
      */
-    private fun dbUserToDataprocMasterVmName(dbUser: DbUser): String {
-        return dbUser.runtimeName + DATAPROC_MASTER_NODE_SUFFIX
-    }
-
-    /**
-     * Returns the standard VM name for a researcher GCE instance assigned by AoU. Leo uses the
-     * exact runtime name for this purpose.
-     */
-    private fun dbUserToGceVmName(dbUser: DbUser): String {
+    private fun dbUserToVmPrefix(dbUser: DbUser): String {
```

Comment:
nit: I would probably inline this and add the doc string to the `runtimeName` property on DbUser.

Reply:
Agreed on inlining - I kept the docs here, otherwise it's not clear why I'm comparing a vm_prefix to a runtime_name - these are sort of separate concerns.